applications/luci-app-statistics/htdocs/luci-static/resources/view/statistics/plugins/users.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+'require baseclass';
+'require form';
+
+return baseclass.extend({
+	title: _('Users Plugin Configuration'),
+	description: _('The users plugin collects statistics about users logged in locally via shell. NOTE: Local shell (wtmp) tracking is NOT enabled in default builds. Additional setup is required to get non-zero counts.'),
```

Comment:
Can you please break the line? It's just too long.

Reply:
@mrjoel